manifests/init.pp

```diff
@@ -27,6 +27,8 @@
   $arch             = $sonarqube::params::arch,
   $https            = {},
   $ldap             = {},
+  # ldap and pam are mutually exclusive. Setting $ldap will annihilate the setting of $pam
```

Comment:
If you look in the template file, I use an `elsif`. If you have some `ldap` setting you will never get the `pam` one. I believe this is more secure. The comment was meant to clarify this behavior.